### PR TITLE
Volume control

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

File: android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java [hunk 1]
```diff
@@ -192,6 +197,9 @@ public void onMethodCall(MethodCall call, Result result) {
       if (call.method.equals("setLooping")) {
```
Comment (mravn-google) [inline android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java]: We should be able to do a `switch` here.

File: android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java [hunk 2]
```diff
@@ -192,6 +197,9 @@ public void onMethodCall(MethodCall call, Result result) {
...
         player.setLooping((Boolean) call.argument("looping"));
         result.success(null);
+      } else if (call.method.equals("setVolume")) {
+        player.setVolume((Double)call.argument("volume"));
```
Comment (mravn-google) [inline android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java]: Add space after cast.

File: ios/Classes/VideoPlayerPlugin.m [hunk 3]
```diff
@@ -294,6 +298,9 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
     } else if ([@"setLooping" isEqualToString:call.method]) {
         [player setIsLooping:[argsMap objectForKey:@"looping"]];
         result(nil);
+    } else if ([@"setVolume" isEqualToString:call.method]) {
+        [player setVolume:[[argsMap objectForKey:@"volume"] doubleValue]];
```
Comment (mravn-google) [inline ios/Classes/VideoPlayerPlugin.m]: Indentation is inconsistent across arms of the `if`-`else`

File: android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java (hunk 1 above)
Comment (sigurdm) [reply android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java]: Done

File: android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java (hunk 2 above)
Comment (sigurdm) [reply android/src/main/java/io/flutter/videoplayer/VideoPlayerPlugin.java]: Done

File: ios/Classes/VideoPlayerPlugin.m (hunk 3 above)
Comment (sigurdm) [reply ios/Classes/VideoPlayerPlugin.m]: Done

**Pull request merged**